Sources/LiveViewNative/Utils/ShapeStyle/AngularGradient.swift

```diff
@@ -49,7 +49,7 @@ extension AngularGradient: Decodable {
         case gradient
         case center
         case angle
-        case startAngle = "start_angle"
-        case endAngle = "end_angle"
+        case startAngle
```

Comment:
how does this look in the LVN template?

Reply:
@bcardarella It's always snake case in the LVN templates. The Swift client is doing an automatic conversion from snake to camel case. That conversion allows us to take advantage of Swift capabilities to generate boilerplate parsing code that we won't have to write anymore.

Reply:
Ah ok, thank you for the explanation